app/channels/photos_channel.rb

```diff
@@ -0,0 +1,10 @@
+class PhotosChannel < ApplicationCable::Channel
+  def subscribed
+    # current_user is always nil here
+    stream_from "photos_for_#{current_user.id}"
```

Comment:
@edance same here as above. `current_user` here is nil and I'm not sure how to grab the authenticated user here.